dev/how-to/deploy/orchestrated/ansible.md

```diff
@@ -130,38 +130,18 @@ Make sure you have logged in to the Control Machine using the `root` user accoun
 
 ## Step 3: Download TiDB Ansible to the Control Machine
 
-1. Log in to the Control Machine using the `tidb` user account and enter the `/home/tidb` directory. The relationship between the `tidb-ansible` version and the TiDB version is as follows:
+Log in to the Control Machine using the `tidb` user account and enter the `/home/tidb` directory. Run the following command to download TiDB Ansible from the master branch of [TiDB Ansible project](https://github.com/pingcap/tidb-ansible). The default folder name is `tidb-ansible`.
 
-    | TiDB version | tidb-ansible tag | Note |
-    | :-------- | :---------------- | :--- |
-    | 2.0 version | v2.0.10, v2.0.11 | It is the latest 2.0 stable version which can be used in the production environment. |
-    | 2.1 version | v2.1.1 ~ v2.1.8 | It is the latest 2.1 stable version which can be used in the production environment (recommended). |
-    | 3.0 version | v3.0.0-beta, v3.0.0-beta.1 | It is currently a beta version which is not recommended to use in the production environment. |
-    | `master` branch | None | It includes the newest features and is updated on a daily basis, so it is not recommended to use it in the production environment. |
-
-2. Download the [corresponding TiDB Ansible versions](https://github.com/pingcap/tidb-ansible/tags) from the [TiDB Ansible project](https://github.com/pingcap/tidb-ansible). The default folder name is `tidb-ansible`.
-
-    > **Note:**
-    >
-    > It is required to use the corresponding tidb-ansible version when you deploy and upgrade the TiDB cluster. If you deploy TiDB using a mismatched version of tidb-ansible (such as using tidb-ansible v2.1.4 to deploy TiDB v2.1.6), an error might occur.
-
-    - Download the tidb-ansible version with a specified tag:
-
-        ```
-        $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
-        ```
-
-    - Download the tidb-ansible version that corresponds to the `master` branch of TiDB:
-
-        ```
-        $ git clone https://github.com/pingcap/tidb-ansible.git
-        ```
+```
+$ git clone https://github.com/pingcap/tidb-ansible.git
+```
 
-    > **Note:**
-    >
-    > It is required to download `tidb-ansible` to the `/home/tidb` directory using the `tidb` user account. If you download it to the `/root` directory, a privilege issue occurs.
+> **Note:**
+>
+> - Deploying and upgrading TiDB clusters requires the corresponding version of `tidb-ansible`. Otherwise, only modifying the version in the `inventory.ini` file might cause errors.
```

Comment:
- The 1st sentence is not clear to me. Please try to understand the meaning and then convey it in a user-friendly way.
- I think the "Otherwise" is a little bit weird.